src/test/scala/advancepassengerinfo/importer/processor/DqFileProcessorImplTest.scala

```diff
@@ -211,6 +215,65 @@ class DqFileProcessorTest extends TestKit(ActorSystem("MySpec"))
       probe.expectMsg(ManifestCall(jsonFileName, manifest))
       probe.expectMsg(JsonFileCall("c", jsonFileName, successful = true, dateIsSuspicious = false))
       probe.expectMsg(ZipFileCall("c", successful = true))
+
+    }
+
+    "handle exceptions while getting nextFiles and recovery to continue" in {
```

Comment:
This test still doesn't show that the code continues after the exception. It shows it correctly processing files 1 & 2, but doesn't look for 3.zip